### PR TITLE
Mpuncel/process exit integration test

### PR DESCRIPTION
**Pull request by mpuncel**

### CODE REVIEW

**Comment by mpuncel** [comment]
This PR has commits from #699 so review that first and then i'll rebase

**Comment by mpuncel** [comment]
found the problem, my vagrant machine has /bin/timeout but travis does not! Removed that from the integration test preparer's config since it's irrelevant to the test!

File: pkg/preparer/setup.go [hunk 1]
```diff
@@ -397,6 +397,7 @@ func New(preparerConfig *PreparerConfig, logger logging.Logger) (*Preparer, erro
 		}
 
 		finishExec = preparerConfig.PodProcessReporterConfig.FinishExec()
+
```
Comment (rudle) [inline pkg/preparer/setup.go]: This commit can go away.

File: integration/single-node-slug-deploy/check.go [hunk 2]
```diff
@@ -98,35 +98,77 @@ func main() {
 		log.Fatalf("Could not schedule RCTL server: %s", err)
 	}
 
-	// Schedule a "legacy" hello pod using a replication controller
-	rcID, err := createHelloReplicationController(tempdir)
-	if err != nil {
-		log.Fatalf("Could not create hello pod / rc: %s\n", err)
+	// Now we're going to test some conditions that each take non-negligible amouhnt of time to verify.
```
Comment (rudle) [inline integration/single-node-slug-deploy/check.go]: typo: amouhnt

**Comment by mpuncel** [comment]
merging on green

**Pull request merged**